lms/djangoapps/teams/views.py

```diff
@@ -59,6 +62,25 @@
 MAXIMUM_SEARCH_SIZE = 100000
 
 
+@receiver(pre_save, sender=CourseTeam)
+def team_pre_save_callback(sender, instance, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Slightly disorganized thoughts: this is a pre-save callback, but do we have any guarantee that the model is actually saved? I haven't checked the source yet, but I'm guessing that Django does its best to ensure that pre-save callbacks are only called when the model will actually get saved. Still, if an exception were thrown somewhere (either in the ORM or by another signal handler which happens to get called after this one), we might emit the event without actually saving the model. I don't immediately see a reason why this couldn't be post-save. Django also gives a slightly nicer API for post-save callbacks by providing a `created` parameter which is `True` if the model is being saved for the first time.